### PR TITLE
doc: Add missing docstrings

### PR DESCRIPTION
**Pull request by ketiltrout**
Also some other doc tweaks

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -95,14 +95,14 @@
 
 # General information about the project.
 project = "ch_util"
-copyright = "2013, CHIME Collaboration"
+copyright = "2013–2024, CHIME Collaboration"
```
Comment (ljgray) [inline doc/conf.py:98]: This could be automated with something like 
```python
datetime.now().year
```

**Review by ljgray — APPROVED** [review]
Looks fine - just one comment

File: doc/conf.py (hunk 1 above)
Comment (ketiltrout) [reply doc/conf.py:98]: It depends on how precise we want to be with this copyright statement.  Automating the ending date implies that we've made copyrightable changes to the documentation every time we push the docs, which probably isn't too unreasonable an assumption.

Comment (ketiltrout) [reply doc/conf.py:98]: I guess ideally, we should go through our doc changes and figure out in which years we actually made copyrightable changes to the docs and only list those years in the copyright statement, but that seems like too much work.

**Pull request merged**